pkg/machines/machines.scss

```diff
@@ -381,3 +381,8 @@ $toast-anim-speed: 200ms;
         margin:  -0.25rem 0.25rem;
     }
 }
+
+/* Add a little vertical space when the list's toolbar wraps */
+.pf-c-toolbar__content-section {
```

Comment:
This needs a bug report upstream.
The documentation also has examples where space is missing in wrapped toolbars.
https://www.patternfly.org/v4/components/toolbar#groups

Reply:
There's a fix pending upstream:

Issue: https://github.com/patternfly/patternfly/issues/3348
PR: https://github.com/patternfly/patternfly/pull/3629